lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -8,66 +8,50 @@
   position: relative;
   display: inline-block;
   &.is-active .dropdown-menu {
-    display: block;
+    opacity: 1;
```

Comment:
So you generally can't use opacity alone to do UI's like this. The reason being I can't click anything behind the dropdown, even when it's closed. Also, you can tab into the hidden menu and focus on the links that are invisible. 

If we want to do an animation, I would suggest using `visibility: hidden` _and_ opacity together. There is a specific way you can combine them to get the animation and still have normal hiding. We do it in modals:

https://github.com/Esri/calcite-web/blob/master/lib/sass/calcite-web/patterns/_modal.scss#L21-L36

If we go that direction, a small transition down during the fade in would help relate the menu with the dropdown via the animation